commands/member.js

```diff
@@ -31,13 +32,14 @@ exports.role = function(message, args){
 
 exports.rmrole = function(message, args){
     let role = message.guild.roles.find(`name`, args.join(' '));
+    logger.info(`User ${message.author.username} leaving role ${role}`);

```

Comment:
Maybe change to 'attempting to leave role' since this is before the role exist check